Tests/Marketplace/upload_packs.py

```diff
@@ -480,7 +480,8 @@ def print_packs_summary(packs_list):
     successful_packs = [pack for pack in packs_list if pack.status == PackStatus.SUCCESS.name]
     skipped_packs = [pack for pack in packs_list if
                      pack.status == PackStatus.PACK_ALREADY_EXISTS.name
-                     or pack.status == PackStatus.PACK_IS_NOT_UPDATED_IN_RUNNING_BUILD.name]
+                     or pack.status == PackStatus.PACK_IS_NOT_UPDATED_IN_RUNNING_BUILD.name
+                     or pack.status == PackStatus.FAILED_DETECTING_MODIFIED_FILES.name]
```

Comment:
FAILED_DETECTING_MODIFIED_FILES happens only if there is now hash? and if so, what will update the hash eventually?

Reply:
@yuvalbenshalom  It happens when not updated build is trying to search more updated build hash. The updated hash will remain. The build should not fail in this case